src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random rand = new Random();
```

Comment:
Avoid using class-level fields when local variables will suffice. The `rand` field can be made into a local variable within the `getRandomBall` method, as it does not need to be shared across multiple methods or instances.